### ISSUE TITLE
Tests failing for autocomplete functional javascript tests

### ISSUE THREAD

**Issue by chriswales95** [localgovdrupal/localgov_elections#47]
Not sure why but one test has started to fail for the autocomplete functional js test. It's not clear why because it runs fine locally.

An example:
https://github.com/localgovdrupal/localgov_elections/actions/runs/9647224005/job/26605613767

It seems to be failing in the setUp method when the user is logged in.

We may have to turn off this test if it doesn't fix itself. I wonder if something upstream has changed somewhere. 

I'm also curious to see if other LGD projects are affected. It's only started in the last few days though and not many projects with functional js tests seem to have been ran lately so can't compare it at the moment.

**Comment by chriswales95** [comment]
It's an issue in other repos too. See here: https://github.com/localgovdrupal/localgov_directories/actions/runs/9647981411/job/26608177211

**Comment by millnut** [comment]
Could you try setting a different admin theme, gin is breaking things at the moment see https://github.com/localgovdrupal/localgov/pull/732 and let me know if that makes a difference

The other modules https://github.com/localgovdrupal/localgov_alert_banner, https://github.com/localgovdrupal/localgov_forms/actions, https://github.com/localgovdrupal/localgov_news/actions and seem to be fine https://github.com/localgovdrupal/localgov_step_by_step/actions

There is also a composer patch not applying at the moment, it may not be the cause but it is a possibility https://github.com/localgovdrupal/localgov_services/issues/263

I know Drupal 10.3 was released last week so maybe that is related, what Drupal core are you running locally when the tests pass?

**Comment by chriswales95** [comment]
@millnut I don't think the functional js test in the elections repo that is failing is using gin. https://github.com/localgovdrupal/localgov_elections/blob/1.x/modules/localgov_elections_constituency_provider/tests/src/FunctionalJavascript/AutocompleteTest.php#L19

It seems to be using stark.

Locally, my core version is 10.2.6. 

I might give it a go with gitpod or something later to see how the tests go on that environment.

**Comment by finnlewis** [comment]
I'll try in GitPod now - I've got one running

**Comment by finnlewis** [comment]
I can replicate this locally. And on Gitpod.

**Comment by millnut** [comment]
Thanks @chriswales95 @finnlewis what Drupal version are you using? 10.3 was released last week and that is what the pipelines use so wonder if it is something specific there. I may have time this evening to try locally and see what the actual 500 error is.

**Comment by millnut** [comment]
@chriswales95 @finnlewis I've looked into this in some depth and this is not due to shared workflows but due to a change in `mink-selenium2-driver` in Drupal 10.3 which requires a different setup for testing.

```
FAILURE - Drupal 10.3
behat/mink (v1.11.0)
lullabot/mink-selenium2-driver (v1.7.2)
behat/mink-browserkit-driver (v2.2.0)

SUCCESS - Drupal 10.2
behat/mink (v1.11.0)
behat/mink-selenium2-driver (v1.7.0)
behat/mink-browserkit-driver (v2.2.0)
```

The same 500 error can be found here https://www.drupal.org/project/drupal/issues/3240792 and specifically https://www.drupal.org/project/drupal/issues/3240792#comment-15481821.

It looks like some changes will be required for the drupal-container image used for the chromedriver which is used in FunctionalJavascript tests. I've created a draft PR to take a look at this for PHP 8.3 branch only https://github.com/localgovdrupal/drupal-container/pull/33 and tests there seem to be running correctly now and fail on a different test and no longer causing error 500. 

So once PHP 8.1 and PHP 8.2 branches for `localgovdrupal/drupal-container` are updated as well we can push these to docker hub and the tests should all work again.
